src/input/JoystickInputDriver.h

```diff
@@ -35,11 +35,13 @@ class JoystickInputDriver : public InputDriver
     void run() override;
     bool open() override;
     void close() override;
-
     const std::string & get_name() const  override;
+    void setJoystickNr(std::string jnr);
+
 private:
     int fd;
     int version;
+    std::string nr="0";
```

Comment:
Prefer uniform initialization:
`std::string nr{"0"};`

Reply:
thank you - I did not know that syntax.